src/core/sock/sockinfo_tcp.h

```diff
@@ -304,6 +353,7 @@ class sockinfo_tcp : public sockinfo, public timer_handler {
         return m_p_connected_dst_entry ? m_p_connected_dst_entry->get_ring() : nullptr;
     }
 
+    void rx_add_ring_cb(ring *p_ring);
```

Comment:
Please specify 'override' so it's clear for the reader that this is a virtual method.

Reply:
I did it first, but then clang start complaining about all the overridden methods in the header. Should I put `override` to the all methods in this header as part of current PR?

Reply:
Not a must. I will do it as part of cache work i hope.